tally/TallyManager.cpp

```diff
@@ -244,20 +244,121 @@ Tally *TallyManager::createTally(unsigned int tally_id,
   }
 
   switch (particle) {
-  case 1:
+  case  1:
```

Comment:
Or maybe, add an mcnp_ipt function to the particle class in pyne, which we use (the class) elsewhere in the codebase.